src/main/java/seedu/address/logic/parser/DisplayCommandParser.java

```diff
@@ -20,14 +23,62 @@
     public DisplayCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args);
+        List<String> argList = argMultimap.getAllValues(EMPTY_PREFIX);
+        String lastArgument = extractLastArgument(argList);
 
         Index index;
         try {
+            if (hasParameterValue(argList)) {
+                if (isWholeNumber(lastArgument)) {
+                    Integer.parseInt(lastArgument);
+                }
```

Comment:
issit possible to extract this double if block into a method of its own? May affect code quality if we leave it like this